src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -186,6 +186,9 @@ private RowMapBuffer getTransactionRows() throws Exception {
 				case XID:
 					buffer.setXid(event.xidData().getXid());
 
+					// feed metric gauge.
+					replicationLag = event.getEvent().getHeader().getTimestamp() - (System.currentTimeMillis() / 1000);
```

Comment:
good point. the issue is that `event.getEvent().getHeader().getTimestamp()` is in seconds though. the doc i created actually says it's millis. i'll update it.